assets/js/slackbot.js

```diff
@@ -0,0 +1,274 @@
+const { App } = require("@slack/bolt");
+
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+});
+
+//Examples links for now
+var BLDRChannel = "C03AZ52PR";
```

Comment:
It's generally bad practice to use hard-coded keys like this in your codebase, even if their misuse wouldn't be a huge risk.